mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -132,6 +118,40 @@ def infer_and_parse_json_input(json_input, schema: Schema = None):
         )
 
 
+@experimental
+def _split_data_and_params(json_input):
+    input_dict = _decode_json_input(json_input)
+    data = {k: v for k, v in input_dict.items() if k in SUPPORTED_FORMATS}
+    params = input_dict.get("params", {})
```

Comment:
By using a default value of empty dict - `{}` - and then passing that dict to `model.predict()`, aren't we breaking serving of old models that don't define / accept the `parameters` argument?

Reply:
Nope, if params passed over is None or {} and ParamSchema is None, then the params value is returned directly without raising an exception in `_enforce_params_schema` function.